### PR TITLE
Nar 640 multipe versions taxonomy

### PR DESCRIPTION
**Pull request by lina1101**

### CODE REVIEW

File: importers/platsannons/converter.py [hunk 1]
```diff
@@ -166,6 +177,17 @@ def convert_ad(message):
         'education_level': [],
     }
 
+    # not nice code, will change in future
```
Comment (oskardrenske-ith) [inline importers/platsannons/converter.py:180]: make Todo: with what needs to be changed

File: importers/platsannons/converter.py [hunk 2]
```diff
@@ -138,6 +138,17 @@ def convert_ad(message):
...
```
Comment (oskardrenske-ith) [inline importers/platsannons/converter.py:141]: make Todo: with what needs to be changed

File: importers/platsannons/converter.py [hunk 3]
```diff
@@ -254,32 +276,52 @@ def _build_contacts(kontaktpersoner):
     return appl_contact_list
 
 
-def _set_occupations(annons, message):
+def _set_occupations(annons, message, taxonomy_2):
     if 'yrkesroll' in message:
         log.debug('NB! Env var to use la v2')
```
Comment (oskardrenske-ith) [inline importers/platsannons/converter.py:281]: Importer only uses LA V2 now

File: importers/platsannons/main.py [hunk 4]
```diff
@@ -118,18 +122,26 @@ def _load_and_process_ads(ad_ids, es_index, es_index_deleted):
...
         processed_ads_total = processed_ads_total + len(ad_batch)
 
         log.info(f'Processed ads: {processed_ads_total}/{len_ads}')
 
     return doc_counter
 
 
-def _convert_and_save_to_elastic(raw_ads, es_index, deleted_index):
+def _get_taxonomy_multiple_versions():
+    headers = {
```
Comment (oskardrenske-ith) [inline importers/platsannons/main.py:134]: one line `headers = {...}`

**Review by oskardrenske-ith — APPROVED** [review]

**Review by halge — CHANGES_REQUESTED** [review]
add logging.

File: importers/platsannons/enricher_mt_rest_multiple.py [hunk 5]
```diff
@@ -80,7 +80,7 @@ def enrich(annonser, typeahead=True):
 def get_doc_headline_input(annons):
     sep = ' | '
     # Add occupation from structured data in headline.
-    doc_headline_occupation = annons.get('occupation', {}).get('label', '')
+    doc_headline_occupation = ' '.join([occupation.get('label', '') for occupation in annons.get('occupation', {})])
```
Comment (halge) [inline importers/platsannons/enricher_mt_rest_multiple.py:83]: add log here

**Review by halge — CHANGES_REQUESTED** [review]

**Pull request merged**